WPMedia/fusion-news-theme-blocks
# Cleanup: Removing blocks that are not converted to 2.0 architecture

## Description

Removes all blocks not in the 2.0.3 manifests.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout remove-tests-unconverted-blocks`
2. Run tests and storybook
3. All remaining blocks should pass tests and be viewable in storybook.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
